osu.Game/Users/User.cs

```diff
@@ -144,5 +144,14 @@ public class RankHistoryData
         public Badge[] Badges;
 
         public override string ToString() => Username;
+
+        /// <summary>
+        /// A user instance for displaying locally created system messages.
+        /// </summary>
+        public static readonly User SYSTEM_USER = new User
```

Comment:
This naming style is usually used for constants. I would prefer `User.System`.

Reply:
it's our current code inspection rule. i'm neutral.